src/test/java/seedu/cakecollate/logic/parser/ParserUtilTest.java

```diff
@@ -50,6 +53,57 @@
     private static final String VALID_TAG_2 = "neighbour";
     private static final String VALID_DELIVERY_DATE = "01/01/2022";
 
+    private static final String VALID_INDEX_LIST_WITH_ONE_INDEX = "1";
```

Comment:
Looks really extensive. Maybe you could add in a test for commas in between the numbers + the numbers being out of order. Other than that lgtm